Updated instructions on downloading the MFA Server software

pastperfect opened the pull request:
The web interface has been updates so the page where you get the software is now named "MFA Server":
https://i.imgur.com/wt1RWOo.png

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

pastperfect committed:
Updated instructions on downloading the MFA Server software

The web interface has been updates so the page where you get the software is now named "MFA Server":
https://i.imgur.com/wt1RWOo.png
